acousticbrainz/views/data.py

```diff
@@ -44,21 +46,21 @@ def view_high_level(mbid):
 
 @data_bp.route("/<uuid:mbid>", methods=["GET"])
 def summary(mbid):
-    lowlevel, highlevel, genres, moods, other = get_summary_data(mbid)
-
-    info = _get_track_info(mbid, lowlevel['metadata'])
-
+    summary = get_summary_data(mbid)
+    
+    info = _get_track_info(mbid, summary['lowlevel']['metadata'] if summary['lowlevel'] else None)
     # Tomahawk player stuff
-    if not ('artist' in lowlevel['metadata']['tags'] and 'title' in lowlevel['metadata']['tags']):
+    if not ('artist' in info and 'title' in info):
```

Comment:
Sure, add a ticket if you want. To me, it's a 6 line change to copy this to a new function, in a part of the code you're already working on, which is why I suggested we make the change now.